### DISCUSSION TITLE
Python version

### DISCUSSION THREAD

**Discussion by mat926** [ghostery/adblocker#3686] [Ideas]
Could you please make a library for Python?

**Comment by remusao** [comment]
Hi,

Thanks for reaching out. Unless you find a way to create a Python wrapper around this library, writing a Python adblocker sounds like a totally new and different project and is out of scope for this repository. Maybe you can have a look at this wrapper around Brave's Rust adblocker library in Python: https://github.com/ArniDagur/python-adblock

I hope that helps,
